lib/validation-stream.js

```diff
@@ -9,6 +9,12 @@ var Duplex = require('readable-stream').Duplex;
 
 var XmlParser = require('./xml-parser');
 
+/** @arg {?string} str Input string to check.
+    @return {!boolean} True if input is undefined, null, or entirely whitespace. */
+function isBlank(str) {
+  return !str || !/\S/.test(str);
```

Comment:
isBlank() returns true if the input is undefined, null, or entirely whitespace. Here are some of the test cases I should have added: `isBlank(null)` => true, `isBlank(undefined)` => true, `isBlank(' ')` => true, `isBlank('A')` => false. When I remove the leading `!str ||`, the first two cases fail.